### PR TITLE
add task and tests

### PR DESCRIPTION
**Pull request by vpolets**

### CODE REVIEW

File: src/case-swapping.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+/**
+ * Implement caseSwapping function:
+ *
+ * Given a string, swap the case for each of the letters.
+ *
+ * Examples:
+ * swap "aBc"      `shouldBe` "AbC"
```
Comment (roman-mate) [inline src/case-swapping.js]:
```suggestion
 * swap "aBc"      should return "AbC"
```

File: src/case-swapping.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+ * swap "ABC"      `shouldBe` "abc"
+ * swap "123235"   `shouldBe` "123235"
+ *
+ * @param {string} phrase
+ *
+ * @return {string}
+ */
+function caseSwapping(phrase) {
```
Comment (roman-mate) [inline src/case-swapping.js]: function names typically start with the verb https://mate-academy.github.io/style-guides/javascript.html#naming--camelCase

swapCase

File: src/case-swapping.test.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  expect(swap(''))
+    .toBe('');
+});
+
+test('Should work with space symbol', () => {
+  expect(swap(' '))
+    .toBe(' ');
+});
+
+test('Should work with different characters', () => {
```
Comment (roman-mate) [inline src/case-swapping.test.js]:
```suggestion
test('Should swap case only for letter and ignore special characters', () => {
```

File: src/case-swapping.test.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+const swap = require('./case-swapping');
+
+test('Should return swapped phrase', () => {
+  expect(swap('HelloWorld'))
+    .toBe('hELLOwORLD');
+});
+
+test('Should return swapped long phrase', () => {
+  expect(swap('lfTm704Unwfis-0acqfnO31 7 _ImaU7-,um.0SIia7.55MJ9'))
```
Comment (roman-mate) [inline src/case-swapping.test.js]: this scenario contains special characters, so it's very similar to the last one. Not sure it brings much value. maybe remove it?

File: src/case-swapping.js [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+/**
+ * Implement swapCase function:
+ *
+ * Given a string, swap the case for each of the letters.
+ *
+ * Examples:
+ * swapCase "aBc"      should return "AbC"
+ * swapCase "ABC"      should return "abc"
+ * swapCase "123235"   should return "123235"
```
Comment (mgrinko) [inline src/case-swapping.js]: why didn't you add this example as a test?

**Pull request merged**